.travis.yml

```diff
@@ -16,6 +16,8 @@ env:
     - TERRAFORM_VERSION=0.11.10
     - YAMLLINT_VERSION=1.8.1
     - TERRAFORM_PROVIDER_RKE_VERSION=0.5.0
+    - KUBECTL_VERSION=v1.12.3
+    - HEML_VERSION=v2.10.0
```

Comment:
`HELM_VERSION`

Reply:
Yep... I'll update. Or would you like to send a PR? Sorry for late reply, I was on vacation.